frontends/web/src/components/headerssync/headerssync.tsx

```diff
@@ -1,5 +1,6 @@
+
```

Comment:
This change seems completely unrelated to the chart, so it could have been a separate PR :)

Reply:
Refactored, because in the version where the headerssync was displayed in the sumamrytable.

What is left is  we share the same spinner. Can we keep it in the same PR but different commit?

Reply:
will open a separate PR